### PR TITLE
[INTEL MKL] Threadpool api implementation for concat and fused batchnorm op.

### PR DESCRIPTION
**Pull request by Srini511**

### CODE REVIEW

**Comment by gbaned** [comment]
@Srini511  Can you please resolve conflicts? Thanks!

File: tensorflow/core/kernels/mkl_fused_batch_norm_op.cc [hunk 1]
```diff
@@ -98,7 +92,8 @@ class MklFusedBatchNormFwdPrimitive : public MklPrimitive {
   //   mean_data:     output data buffer of means
   //   variance_data: output data buffer of variances
   void Execute(const T* src_data, const U* weights_data, T* dst_data,
-               U* mean_data, U* variance_data, U* workspace_data) {
+               U* mean_data, U* variance_data,
+               std::shared_ptr<stream> fwd_stream, U* workspace_data) {
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_batch_norm_op.cc:96]: Just curious, why do we put `fwd_stream` in front of `workspace_data`? (`BwdPrimitive` and other PRs have `stream` last.)

**Review by penpornk — COMMENTED** [review]
Thank you for the PR and sorry for the delay!

File: tensorflow/core/kernels/mkl_fused_batch_norm_op.cc (hunk 1 above)
Comment (Srini511) [reply tensorflow/core/kernels/mkl_fused_batch_norm_op.cc:96]: Right. This conflicted with another PR that was merged recently. So I decided to put it this way. Originally fwd_stream was the last.

**Review by penpornk — APPROVED** [review]
Thanks for the clarification.
